### ISSUE TITLE
Timeline does not render events when used as container item

### ISSUE THREAD

**Issue by bmblb** [bryntum/support#2665]
To reproduce on timeline app replace timeline/gantt instantiation with this:
```
const panel = window.panel = new Container({
    appendTo : 'container',

    layoutStyle : {
        flexFlow : 'column'
    },

    items : [
        {
            type : 'timeline',
            ref  : 'timeline',

            height : 300,

            tbar : [
                'Timeline size',
                {
                    type  : 'buttonGroup',
                    items : [
                        {
                            text        : 'Small',
                            toggleGroup : 'size',
                            listeners   : {
                                toggle : setTimelineHeight
                            }
                        }, {
                            text        : 'Medium',
                            toggleGroup : 'size',
                            pressed     : true,
                            listeners   : {
                                toggle : setTimelineHeight
                            }
                        }, {
                            text        : 'Large',
                            toggleGroup : 'size',
                            listeners   : {
                                toggle : setTimelineHeight
                            }
                        }
                    ]
                }
            ],

            project
        },
        {
            type : 'gantt',
            flex : 1,

            columns : [
                { type : 'name', width : 250 },
                { type : 'showintimeline', width : 150 }
            ],
            project
        }
    ]
});

const timeline = panel.widgetMap.timeline;
```

**Comment by bmblb** [comment]
![image](https://user-images.githubusercontent.com/45400762/114732293-12e46c80-9d4b-11eb-8a73-8a9b1f0c39e0.png)

**Comment by bmblb** [comment]
There is a race condition which leads to rows not being initialized by the RowManager. Timeline works in case:
- timeline is instantiated after project is loaded and committed
- timeline is rendered immediately using appendTo/adopt/etc... (which explains why it doesn't work inside container - rendering is postponed)

This particular issue is caused by this flow:
1. project load is started
2. timeline is created but not yet rendered
3. timeline calls `fillFromTaskStore.now()` which suspends refresh and waits for project to commit
4. project commits, fires `refresh` event which is supposed reinitialize rows. but on step 3 refresh was suspended so refresh doesn't occur
5. `commitAsync` promise is resolved and `fillFromTaskStore` unblocks refreshes. but it doesn't reinitialize rows. and there is a flag raised on the timeline: `timeline.horizontal.refreshAllWhenReady` which is supposed to be cleaned up by project refresh listener

So we end up with a timeline hanging there without any rows, waiting for `RowManager.reinitialize` call
